website/docs/r/aws_auth_backend_roletag_blacklist.html.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: "vault"
+page_title: "Vault: vault_aws_auth_backend_roletag_blacklist resource"
+sidebar_current: "docs-vault-resource-aws-auth-backend-roletag-blacklist
+description: |-
+  Configures the periodic tidying operation of the blacklisted role tag entries.
+---
+
+# vault\_aws\_auth\_backend\_roletag\_blacklist
```

Comment:
Nitpick: I know at least in AWS provider-land, the backslashes are no longer required for the documentation header